src/components/molecules/FormFields/URLInput/Input.tsx

```diff
@@ -17,6 +18,20 @@ export default function URLInput({
 }): ReactElement {
   const [field, meta] = useField(props as FieldInputProps<any>)
 
+  const [buttonDisabled, setButtonDisabled] = useState(true)
+
+  useEffect(() => {
+    if (!field?.value) return
+
+    setButtonDisabled(
+      !field?.value ||
+        field.value === '' ||
+        !isUrl(field.value) ||
+        field.value.includes('javascript:') ||
+        meta?.error
```

Comment:
suggestion(non-blocking): When having so many conditions, it might be beneficial to extract this into a separate function to improve readability. As we are dealing with an XSS attack here, the function could be named accordingly or have a comment attached to it.